cas/incident-reporting.md

```diff
@@ -0,0 +1,101 @@
+
+## Objectives and Expectations
+
+The purpose of Incident Reporting is to help us work together to build a more secure web. 
+
+Depending on the root programs in which a CA Owner participates, it may be required to:
+* create [Audit Incident Reports](audit-incident-report) when audits have non-conformities, qualifications, or modified opinions; and/or
+* create [Incident Reports](incident-report) for all other incidents.
+
+These reports provide lessons learned and transparency about the steps the CA Owner takes to address the immediate issue and prevent future issues. If the underlying problem goes unfixed, then other issues that share the same root cause will subsequently surface. Additionally, Incident Reports help the Web PKI ecosystem as a whole because they promote continuous improvement, information sharing, and highlight opportunities to define and adopt improved practices, policies, and controls.
+
+CA Incident Reports are expected to:
+- be candid, timely, and transparent;
+- clearly and objectively communicate the scope, impact, and severity of an incident;
+- help community members understand the relevant architectures, implementations, operations, and external dependencies surrounding an incident;
+- demonstrate a detailed understanding of the root cause(s) of an incident;
+- unambiguously explain how systems, processes, and/or policies failed; 
+- describe the circumstances that prevented the problem(s) from being detected earlier;
+- describe a clear timeline of a CA Owner's actions while responding to and remediating an incident;
+- include a detailed and measurable explanation of actions taken or planned by the CA Owner that demonstrate a substantive commitment of how the CA Owner's systems, policies, and/or processes will be made more robust (i.e., demonstrate continuous improvement); and
+- share lessons learned that could be helpful to all CA Owners in building better systems policies, and/or processes.
+
+### Incident Reporting: Good Practices
+
+1. **Be Detailed and Precise**:
+   - Document, in detail, all findings and steps taken from discovery or initial notification through to remediation.
+   - Include precise timestamps, technical data, and decision rationales.
+   - Share relevant configurations, code snippets, or settings that were implicated, in compliance with security norms (i.e., do not disclose confidential or sensitive information).
+
+2. **Respond Promptly**:
+   - Acknowledge receipt of the incident report quickly.
+   - Provide initial assessments and expected timelines for resolution to keep stakeholders informed.
+   - Respond to comments or questions within one business day.
```

Comment:
Response time here isn't just a "good practice", it's a requirement. Does it weaken the requirement to have it under a "good practice" heading?

Many of these "good practices" actually look like things that should be required, in that if an Incident Report doesn't exhibit them, it should be fixed until it does.